plugins/device/deviceutils/utils.go

```diff
@@ -0,0 +1,41 @@
+package deviceutils
+
+import (
+	"os/exec"
+)
+
+//CreatePV is used to create physical volume.
+func CreatePV(device string) error {
+	pvcreateCmd := exec.Command("pvcreate", "--metadatasize=128M", "--dataalignment=256K", device)
+	if err := pvcreateCmd.Run(); err != nil {
```

Comment:
The [Go report service](https://goreportcard.com/report/github.com/gluster/glusterd2) will likely flag this as redundant. You can directly return.

```go
return pvcreateCmd.Run()
```

Please do the same for all the functions in this file.